docs/AI-for-security/api/bulk-actions-prompts-api.asciidoc

```diff
@@ -0,0 +1,249 @@
+[[bulk-actions-prompts-api]]
+=== Bulk prompts actions
+
+Applies a bulk action to multiple prompts. The bulk action is applied to all prompts that match the filter or to the list of prompts by their IDs.
+
+[discrete]
+===== Request URL
+
+`POST  <kibana host>:<port>/api/security_ai_assistant/prompts/_bulk_action`
+
+[discrete]
+===== Request body
+
+A JSON object with the following properties:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+| `create` | <<bulk-create-object-schema, BulkCreateAction[]>> |  Array of prompts objects to create. | No
+| `delete` | String[] |  Array of prompts IDs to which a bulk action will be applied. | No
+| `update` | <<bulk-update-object-schema, BulkUpdateAction[]>> |  Array of prompts objects with the fields to update. | No
+
+|==============================================
+
+
+[[bulk-create-object-schema]]
+[discrete]
+==== BulkCreateAction object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`name` |String |Prompt short title. |Yes
+|`id` |String |Prompt ID. |Yes
+|`promptType` |String |Prompt type. Could be 'system' or 'quick' |Yes
+|`consumer` |String |Solution application to which prompt belongs. |Yes
+|`content` |String |Prompt text whixh will be send to LLM. |Yes
+|`color` |String |Is only for quick prompts to highligh in the Assistant UI. |No
+|`categories` |String[] |Array of prompts categories. |No
+|==============================================
+
+
+[discrete]
+[[bulk-update-object-schema]]
+===== `BulkUpdateAction` schema
```

Comment:
```suggestion
==== BulkUpdateAction object
```
Making this the same as the header for BulkCreateAction object. They seem to be comparable objects so figured they should have similar headers. Feel free to ignore if that's inaccurate.